upload.go

```diff
@@ -207,3 +212,13 @@ func saveHandler(w http.ResponseWriter, r *http.Request, ignored string) {
 	// a confirmation email has ben send...
 	renderTemplate(w, "save", map[string]string{"Email": email})
 }
+
+func verifyEmail(email string) error  {
+	for _, domain := range emailDomains {
+		if strings.HasSuffix(email, "@" + domain) {
```

Comment:
I think it would be better to make the comparison case-insensitive

Reply:
good point. I changed that with a force push. I hope it is okay. I am not  familiar with githubs workflow.